superset-frontend/packages/superset-ui-chart-controls/test/utils/getColorFormatters.test.ts

```diff
@@ -188,6 +188,22 @@ describe('getColorFunction()', () => {
     expect(colorFunction(150)).toBeUndefined();
   });
 
+  it('getColorFunction BETWEEN_OR_EQUAL without opacity', () => {
+    const colorFunction = getColorFunction(
+      {
+        operator: COMPARATOR.BETWEEN_OR_EQUAL,
+        targetValueLeft: 50,
+        targetValueRight: 100,
+        colorScheme: '#FF0000',
+        column: 'count',
+      },
+      countValues,
+      false,
+    );
+    expect(colorFunction(50)).toEqual('#FF0000');
+    expect(colorFunction(100)).toEqual('#FF0000');
```

Comment:
Would it be worth adding an assertion in the middle of the range (e.g. 75) for good measure?

I'm also not sure if there's another test that asserts against values outside the range (e.g. 25 or 125 in this case). If not, that might be good to add in this test or a new one.

Reply:
added tests, please resolve it ok